tracer/test/Datadog.Trace.OpenTracing.Tests/Datadog.Trace.OpenTracing.Tests.csproj

```diff
@@ -13,9 +13,18 @@
 
   <ItemGroup Condition=" !$(TargetFramework.StartsWith('net4')) ">
     <PackageReference Include="System.Net.Http" Version="4.3.4" />
+
+    <!-- These are only required because we load Datadog.Trace using reflection  -->
```

Comment:
I don't really understand, seeing as we reference Datadog.Trace that also has these, but whatevs

Reply:
Because `Datadog.Trace` also pulls them in with `PrivateAssets="All"`?

Reply:
🤦‍♂️ Oh yeah, so they do, thanks 😅